### DISCUSSION TITLE
Add file access authorization rules to the sync server protocol

### DISCUSSION THREAD

**Discussion by np5** [google/santa#1040] [Ideas]
Currently, the file access watch items are loaded from a plist stored on disk. It would be great to be able to managed them with a sync server.

Maybe the rule download phase of the synchronization could be expanded to include watch items?

**Comment by pmarkowsky** [comment]
This seems pretty reasonable. I think we need to determine somethings first though.

1. We need to have  dialogue box for File access authorization blocks. (This should probably happen around the 2023.3-2023.4) timeline.
2. We need to add the ability to upload file accesses / blocks to the sync server.
3. We need to figure out how to store the rules in Santa (e.g. do we create a separate table in the db or do we encode the rule in the identifier column)?
